doc/man1/openssl-cms.pod.in

```diff
@@ -462,16 +440,9 @@ portion of a message so they may be included manually. If signing
 then many S/MIME mail clients check the signers certificate's email
 address matches that specified in the From: address.
 
-=item B<-attime>, B<-check_ss_sig>, B<-crl_check>, B<-crl_check_all>,
-B<-explicit_policy>, B<-extended_crl>, B<-ignore_critical>, B<-inhibit_any>,
-B<-inhibit_map>, B<-no_alt_chains>, B<-no_check_time>, B<-partial_chain>, B<-policy>,
-B<-policy_check>, B<-policy_print>, B<-purpose>, B<-suiteB_128>,
-B<-suiteB_128_only>, B<-suiteB_192>, B<-trusted_first>, B<-use_deltas>,
-B<-auth_level>, B<-verify_depth>, B<-verify_email>, B<-verify_hostname>,
-B<-verify_ip>, B<-verify_name>, B<-x509_strict>
+{- $OpenSSL::safe::opt_v_item -}
 
-Set various certificate chain validation options. See the
-L<openssl-verify(1)> manual page for details.
+Any verification errors cause the command to exit.
```

Comment:
Perhaps not a show-stopper, but I think better would be:
```
Certificate verification terminates with a failure as soon as the first error detected. Additional errors if any are not reported.
```

Reply:
Should something similar be said in the `ocsp(1)` manpage?

Reply:
I agree that the wording should be the same across the manpages.  I prefer mine because it says that the *program* exits, while your suggestion says *verification terminates* without saying the effect on the program.

Reply:
But the program does not exit at that point, it merely returns verification failure to the caller, with the caller doing whatever it does when verification fails (which may be exit, but I did not check). If you're sure that the effect is effectively an immediate exit, I'll grudgingly accept the less precise description. :-)

Reply:
Yes, verify will exit if it's check() routine fails, CMS exits on verification failure, OCSP same, etc.  Since the "caller" is the application program, and the manpage is describing the application program, I think it's correct.